include/net/nrf_cloud_rest.h

```diff
@@ -50,16 +49,25 @@ enum nrf_cloud_rest_agps_req_type {
 	NRF_CLOUD_REST_AGPS_REQ_UNSPECIFIED,
 };
 
+#define NRF_CLOUD_REST_TIMEOUT_MINIMUM		(5000)
```

Comment:
How did you arrive at 5000?

Reply:
actually, after some additional testing i think the timeout for the http_client should just not be used.
when the timeout expires it just closes the socket, which does not cause the [recv call](https://github.com/nrfconnect/sdk-zephyr/blob/a64e96d17cc741937a8f5ff01a824ef0df22f662/subsys/net/lib/http/http_client.c#L418) to exit and eventually a fault occurs 😑 

updated.

Reply:
That was my "fault" before fix 😁